### ISSUE TITLE
Q：for t in range(max(decode_lengths)):

### ISSUE THREAD

**Issue by kangkang59812** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#69]
![image](https://user-images.githubusercontent.com/12228641/56659238-29f9a200-66cf-11e9-83be-bb29265905ec.png)
When I debug 'train.py', it always skip 'preds = self.fc(self.dropout(h))' and the codes blow it.
But they are all in 'for' cycle, what happend?
